journal-of-urban-technology.csl

```diff
@@ -0,0 +1,318 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Journal of Urban Technology</title>
+    <title-short>JUT</title-short>
+    <id>http://www.zotero.org/styles/journal-of-urban-technology</id>
+    <link href="http://www.zotero.org/styles/journal-of-urban-technology" rel="self"/>
+    <link href="http://www.zotero.org/styles/hiob-ludolf-centre-for-ethiopian-studies" rel="template"/>
+    <link href="https://www.tandfonline.com/action/authorSubmission?show=instructions&amp;journalCode=cjut20&amp;#style" rel="documentation"/>
+    <link href="https://www.tandf.co.uk//journals/authors/style/layout/style_cjut.pdf" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="engineering"/>
+    <issn>1063-0732</issn>
+    <eissn>1466-1853</eissn>
+    <updated>2020-03-27T15:30:17+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en-GB">
```

Comment:
@POBrien333, can you check the locale section here? If the style gives correct output, I assume this locale section can be removed as it's specific to "en-GB", while the style is locked to "en-US".